packages/config/config/devices/0x010f/fgwdeu.json

```diff
@@ -310,9 +300,31 @@
 			"minValue": 0,
 			"maxValue": 15,
 			"defaultValue": 0,
-			"readOnly": true,
+			"readOnly": false,
 			"writeOnly": false,
-			"allowManualEntry": true
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "no scenes",
+					"value": 0
+				},
+				{
+					"label": "Key pressed once",
+					"value": 1
+				},
+				{
+					"label": "Key pressed twice",
+					"value": 2
+				},
+				{
+					"label": "Key pressed 3 times",
+					"value": 4
+				},
+				{
+					"label": "Key held down and released",
+					"value": 8
+				}
+			]
```

Comment:
These options should be modeled as partial parameters so they can be combined with each other. The other parameters that have the same options aren't correct either.